.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Release
+on:
+  # Runs automatically when a tag beginning with 'release-' is pushed.
+  push:
+    tags:
+      - release-*
```

Comment:
With Boulder we do dated releases; for unbound_exporter we do versioned releases (https://github.com/letsencrypt/unbound_exporter/releases). So let's make this trigger on tags matching `v*`.